src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Option.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import React from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import optionStyles from './option.scss';
+
+const SELECTED_ICON = 'check';
+
+export default class Option extends React.PureComponent {
+    props: {
+        selected: boolean,
+        disabled: boolean,
+        focus: boolean,
+        value: string,
+        children?: string,
+        onClick?: (s: string) => void,
+    };
+
+    static defaultProps = {
+        disabled: false,
+        selected: false,
+        focus: false,
+        value: '',
+    };
+
+    item: HTMLElement;
+    button: HTMLButtonElement;
+
+    componentDidMount() {
+        if (this.props.focus) {
+            setTimeout(() => {
```

Comment:
The button is always set in `componentDidMount`. The issue is that it's not ready to receive the focus in the dom. Moreover the `ref` callback gets called before the `componentDidMount`. It works in the `ref` but also only with `setTimeout`